great_expectations/data_context/data_context/serializable_data_context.py

```diff
@@ -52,6 +52,7 @@ class SerializableDataContext(AbstractDataContext):
     ]
     GX_DIR = "great_expectations"
     GX_YML = "great_expectations.yml"
+    ZEP_YAML: pathlib.Path = pathlib.Path("zep_config.yaml")
```

Comment:
Followup PR will allow for zep config and the original config to live in the same yaml file.